PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/ui/SignInWindow.java

```diff
@@ -220,30 +229,29 @@ public void run(ProgressIndicator indicator) {
     private void doSignOut() {
         try {
             accountEmail = null;
-            AdAuthManager.getInstance().signOut();
+            // AuthMethod.AD is deprecated.
+            getAuthMethodManager().getAdAuthManagerByAuthMethod(AuthMethod.DC).signOut();
```

Comment:
Same as above, could we call singout of getAuthMethodManager() directly?

Reply:
Looks like the current method is only for DC auth.